### PR TITLE
Issue 8 support us pages

### LINKED ISSUE #8: Support us pages
# Basic Information
Create the [Support Us pages](https://www.starick.org.au/archived/who-we-are-v3/) on the site

# Other Information
Paypal donation component will be provided by #16. There are several subpages for this, so it may be worth splitting into multiple subissues.

### PR DESCRIPTION
**Pull request by MoeBuTa**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related issue

- Resolve #8

### CODE REVIEW

File: src/components/Services/ResourceAndToolkits/child-safe-list.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  }>;
+}
+
+export default function ChildSafeList({ itemList }: ChildSafeListProps) {
+  return (
+    <div>
+      {itemList.map((child, index) => (
+        <ChildSafeItem
+          key={index}
+          backgroundColor={child.backgroundColor} // Assuming order is a number
```
Comment (cjnbennett) [inline src/components/Services/ResourceAndToolkits/child-safe-list.tsx]: Strange comment here?

**Review by cjnbennett — CHANGES_REQUESTED** [review]
Looks good, just a couple minor things - see the comment I made on the files, and also it looks like the PayPal button has too high a z-index and will appear on top of the navigation bar (if you scroll that far).

File: src/components/Services/ResourceAndToolkits/child-safe-list.tsx (hunk 1 above)
Comment (MoeBuTa) [reply src/components/Services/ResourceAndToolkits/child-safe-list.tsx]: fixed these two issues

**Review by cjnbennett — APPROVED** [review]
Lgtm

**Pull request merged**